programming-basics/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "exercises",
+  "version": "1.0.0",
+  "description": "Exercícios da PodCodar",
+  "main": "README.md",
+  "scripts": {
+    "test": "jest $INIT_CWD/test.js",
+    "test:all": "jest"
```

Comment:
**minor**: Please, double check if this script `test:all` will run. It might have stopped to work, since we've change the root folder.